numpy/testing/tests/test_utils.py

```diff
@@ -609,8 +609,8 @@ def setup(self):
         self._assert_func = assert_approx_equal
 
     def test_simple_arrays(self):
```

Comment:
Since this was never intended to be used on 1d arrays:
```suggestion
     def test_simple_0d_arrays(self):
```